DOCS.md

```diff
@@ -114,6 +115,43 @@ To use `$${IMAGE_VERSION}` or `$IMAGE_VERSION`, see the [Drone docs][environment
 [expand]: https://golang.org/pkg/os/#ExpandEnv
 [environment]: http://docs.drone.io/environment/
 
+## Using "extra" `kubectl` versions
+
+### tl;dr
+
+To run `drone-gke` using a different version of `kubectl` than the default, set `kubectl-version` to the version you'd like to use.
+
+For example, to use the **1.14** version of `kubectl`:
+
+```yml
+image: nytimes/drone-gke:0.9.1
+kubectl_version: "1.14"
+```
+
+This will configure the plugin to execute `/google-cloud-sdk/bin/kubectl.1.14` instead of `/google-cloud-sdk/bin/kubectl` for all `kubectl` commands.
+
+### Background
+
+Beginning with the [`237.0.0 (2019-03-05)` release of the `gcloud` SDK](https://cloud.google.com/sdk/docs/release-notes#23700_2019-03-05), "extra" `kubectl` versions are installed automatically when `kubectl` is installed via `gcloud components install kubectl`.
+
+These "extra" versions are installed alongside the SDK's default `kubectl` version at `/google-cloud-sdk/bin` and are named using the following pattern:
+
+```sh
+kubectl.$clientVersionMajor.$clientVersionMinor
+```
+
+To list all of the "extra" `kubectl` versions available within a particular version of `drone-gke`, you can run the following:
+
+```sh
+# list "extra" kubectl versions available with nytimes/drone-gke:0.9.1
+docker run \
+  --rm \
+  --interactive \
+  --tty \
+  --entrypoint '' \
+  nytimes/drone-gke:0.9.1 list-extra-kubectl-versions
```

Comment:
todo(@montmanu):

- [ ] replace `0.9.1` with the correct version once released

Reply:
The image tag probably isn't necessary unless desired, then no need to update it. Or use the version that this became available.